src/Shapes.cpp

```
@@ -26,7 +26,23 @@ void Point::draw() {
 // TODO: create the shape
 
 void Circle::draw() {
+
+    const BaseColor bColor = _color.toBase();
+    glColor3f(bColor.r, bColor.g, bColor.b);
     
+    const int N = 32;
```

Clang (14.0.6)

[warning] variable name 'N' is too short, expected at least 3 characters

(readability-identifier-length)